src/Core/Framework/Script/Api/AclFacade.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Script\Api;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Log\Package;
+
+#[Package('framework')]
+class AclFacade
+{
+    public function __construct(private readonly Context $context)
```

Comment:
Nitpicking here: Does it make sense to rename the var as `$appContext`? I needed to double check if this will be the general request context or something app specific and only then saw the factory code

Reply:
makes sense, it's also a bit weird for me that `AppContextCreator` lives in the DAL namespace 😬

Reply:
it is internal, so move it?